src/zig_clang.cpp

```diff
@@ -1985,6 +1985,24 @@ unsigned ZigClangFunctionDecl_getAlignedAttribute(const struct ZigClangFunctionD
     return 0;
 }
 
+bool ZigClangVarDecl_getPackedAttribute(const struct ZigClangVarDecl *self) {
+    auto casted_self = reinterpret_cast<const clang::VarDecl *>(self);
+    if (const clang::PackedAttr *PA = casted_self->getAttr<clang::PackedAttr>()) {
```

Comment:
I'm not sure if I am following the proper steps to build, but I get unused variable warnings:

```
/home/ionspin/ZigSource/zig/src/zig_clang.cpp: In function ‘bool ZigClangVarDecl_getPackedAttribute(const ZigClangVarDecl*)’:
/home/ionspin/ZigSource/zig/src/zig_clang.cpp:1990:34: error: unused variable ‘PA’ [-Werror=unused-variable]
 1990 |     if (const clang::PackedAttr *PA = casted_self->getAttr<clang::PackedAttr>()) {
      |                                  ^~
/home/ionspin/ZigSource/zig/src/zig_clang.cpp: In function ‘bool ZigClangFieldDecl_getPackedAttribute(const ZigClangFieldDecl*)’:
/home/ionspin/ZigSource/zig/src/zig_clang.cpp:1999:34: error: unused variable ‘PA’ [-Werror=unused-variable]
 1999 |     if (const clang::PackedAttr *PA = casted_self->getAttr<clang::PackedAttr>()) {
      |                                  ^~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/zigcpp.dir/build.make:104: CMakeFiles/zigcpp.dir/src/zig_clang.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:121: CMakeFiles/zigcpp.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

Reply:
Those variables are indeed unused. I will push a fix.